src/components/loadable.cjsx

```diff
@@ -21,14 +23,17 @@ module.exports = React.createClass
     renderError: React.PropTypes.func.isRequired
     renderBug: React.PropTypes.func.isRequired
 
+  getInitialState: ->
+    refreshButton: false
+
   getDefaultProps: ->
 
     # Enables a renderStatus prop function with a component other than a div
-    renderLoading: ->
-      <div className='-loading'>Loading...</div>
+    renderLoading: (refreshButton) ->
```

Comment:
From talking with @philschatz, when the loading times out (for example after the 30 seconds delay being used for the delayed button), should it just renderError()? If that's the case, then the renderLoading would not have to consider the button at all.